src/components/Comments/Comment.js

```diff
@@ -215,6 +295,7 @@ const Comment = ({
       </View>
 
       <BlockComponent ref={refBlockComponent} screen={'feed_comment_item'} />
+      <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} height={182} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BottomSheetMenu` component is correctly added and passed the necessary props. It's important to ensure that this component is implemented correctly and handles all edge cases.
```diff
+ <BottomSheetMenu refSheet={refSheet} dataSheet={dataSheet} height={182} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->